src/site/stages/build/plugins/download-drupal-assets.js

```diff
@@ -38,8 +38,8 @@ async function downloadFile(
           }. ${e} Retries remaining: ${retries}`,
         );
         // Pause to give the proxy connection a break.
-        // eslint-disable-next-line no-await-in-loop
-        await new Promise(resolve => setTimeout(500, resolve));
+        // eslint-disable-next-line no-await-in-loop,no-loop-func
+        await new Promise(resolve => setTimeout(resolve, 2000 - retries * 500));
```

Comment:
I'm having a hard time understanding this line. If it were the first retry, `retries` would be 2. So `2000 - (2 * 500))` - we pause for one second. Then on the next lap, we pause for a half second, then on the next no pause at all. How come we need to do it like this? (I could be getting this logic wrong)

Reply:
The pause _increases_ with the number of retries:

```
retries    delay
------    ------
2          2000 - 2 * 500 = 1000
1          2000 - 1 * 500 = 1500
0          2000 - 0 * 500 = 2000
```